docs/index.rst

```diff
@@ -0,0 +1,24 @@
+##########################
+edx-platform Documentation
+##########################
+
+Developer documentation is split between this docs directory and documentation
+available from the `Developer Documentation Index`_ in Confluence.
+
+User documentation can be read on the `Open edX ReadTheDocs`_.  The source for
+this documentation is in the `edx-documentation`_ repository.
+
+You can also check out `Confluence`_, our wiki system. Once you sign up for
```

Comment:
Nit: I would prefer to link to the Open edX Development space: https://openedx.atlassian.net/wiki/display/OpenDev/Open+edX+Development

I also wouldn't include the explanation of what Confluence is and how to use it. I think folks know what a wiki is now.